packages/apify/src/actor.ts

```diff
@@ -165,6 +176,21 @@ export class Actor<Data extends Dictionary = Dictionary> {
      * @ignore
      */
     async init(options: InitOptions = {}): Promise<void> {
+        if (this.initialized) {
+            log.warning(`Actor#init() was called multiple times. This will have no effect.`);
```

Comment:
Hmm, which would you prefer? Should we warn and init only once if users call Actor.init multiple times or leave it as is and allow multiple init calls to work?

Reply:
> This will have no effect

I don't think we need a warning when it literally warns about nothing happening :] 

There are no side effects to this, right? The use case is `Actor.getInput()` as a top level await in two files - you need to run `Actor.init` first (in both files).

Reply:
This warning is now done so we prevent subsequent Actor.inits from overwriting storage clients (see the return after the log). That said...  I'd personally prefer if people did 
```ts
await Actor.init();

const {} = await import('./file-that-requires-actor-init');
```
but I wanted to know what you would prefer